modules/ROOT/pages/tools/io-tool.adoc

```diff
@@ -0,0 +1,110 @@
+= IO-Tool
+:imagesdir: 
+
+== What is the IO-tool?
+
+https://io.my-agrirouter.com/[The agrirouter InOut-Tool] is a tool to simulate a remote endpoint and to learn to communicate via agrirouter. 
+
+image::io-tool/settings.png[]
+
+
+Using the IO-Tool you can 
+
+* exchange data
+* send and receive telemetry data
+* set preferences for the data exchange
+* check the logs and debug
+* simulate end points and set routings
+
+
+== Getting Started
+
+First you need to log in to your account, which can be requested from Agrirouter support (support@my-agrirouter.com).
+
+Here is an email template for you
+
+_Dear Sir/Madam,_
```

Comment:
You can put this template in a block quote, see https://docs.asciidoctor.org/asciidoc/latest/blocks/blockquotes/